kgbrules/status_rules.py

```diff
@@ -0,0 +1,100 @@
+"""Rules applied to the first line of the commit"""
+
+import re
+
+from .config import MAX_STATUS_LENGTH
+from .config import TYPES
+
+STATUS_PATTERN = re.compile(
+    r"^([^()\s]*)(\s*)\(([^()\s]*)\)(\s*):(\s*)(.*\w)(\s*)$")
+
+_TYPE_STRING = ", ".join(sorted(TYPES))
+
+
+def _is_merge(status_line):
+    return status_line.startswith("Merge pull request #")
+
+
+def _matches_pattern(status_line):
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Function has a complexity of 13 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=python%3AFunctionComplexity)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a docstring to this function. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=python%3AS1720)